lib/reporters/base.js

```diff
@@ -292,17 +295,24 @@ exports.list = function (failures) {
 
     // indented test title
     var testTitle = '';
-    test.titlePath().forEach(function (str, index) {
-      if (index !== 0) {
-        testTitle += '\n     ';
-      }
-      for (var i = 0; i < index; i++) {
-        testTitle += '  ';
-      }
-      testTitle += str;
-    });
+    // Incase test is an AggregateError object when recursively listing errors
```

Comment:
[Docs] Nit: I think the `instanceof AggregateError` implies what this comment is trying to say. 🙂 

```suggestion
```

Reply:
Yeah, that one speaks for itself, you're right!